<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenceLabeller</class>
 <widget class="qMRMLWidget" name="SequenceLabeller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>398</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Slice segmentation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input sequence browser:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="segSequenceBrowserSelector">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSequenceBrowserNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Input segmentation:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="segSegmentationSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="segApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Input volume:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="segInputVolumeSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="qMRMLNodeComboBox" name="segOutputSequenceBrowserSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSequenceBrowserNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Output sequence brower:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Markup classification</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Input volume:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="markupVolumeSelector">
        <property name="toolTip">
         <string>Pick the output to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Input markup:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="markupMarkupSelector">
        <property name="toolTip">
         <string>Result with inverted threshold will be written into this volume</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="markupApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Input sequence browser:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="markupSequenceBrowserSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSequenceBrowserNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="qMRMLNodeComboBox" name="markupOutputSequenceBrowserSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSequenceBrowserNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Output sequence browser:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Output label:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="markupsOutputLabelSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLTextNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segInputVolumeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segOutputSequenceBrowserSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segSegmentationSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segSequenceBrowserSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>markupMarkupSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>markupOutputSequenceBrowserSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>markupSequenceBrowserSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>markupVolumeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SequenceLabeller</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>markupsOutputLabelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
